automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -153,6 +157,99 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
+            "methodRepoMethod" -> methodRepoMethod,
+            "name" -> "echo_to_file-configured",
+            "namespace" -> "gatk",
+            "rootEntityType" -> "vcf_file",
+            "prerequisites" -> Map(),
+            "inputs" -> Map("echo_strings.echo_to_file.input1" -> "this.VCF_File_Name"),
+            "outputs" -> Map("echo_strings.echo_to_file.out" -> "workspace.output"),
+            "methodConfigVersion" -> 1,
+            "deleted" -> false,
+            "dataReferenceName" -> snapshotName
+          )
+          Rawls.postRequest(
+            uri = createMethodConfigUrl.toString(),
+            content = createMethodConfigPayload)
+
+          // run analysis on the snapshot
+          val createSubmissionUrl  = Uri(Rawls.url).withPath(Path(s"/api/workspaces/$projectName/$workspaceName/submissions"))
+          // TODO: consider using 'SubmissionRequest' case class when AS-623 is done
+          val createSubmissionPayload = Map(
+            "useCallCache" -> true,
+            "deleteIntermediateOutputFiles" -> false,
+            "methodConfigurationNamespace" -> "gatk",
+            "methodConfigurationName" -> "echo_to_file-configured"
+          )
+          val response = Rawls.postRequest(
+            uri = createSubmissionUrl.toString(),
+            content = createSubmissionPayload)
+          // TODO: AS-622 parse into 'SubmissionReport' case class following its (i.e. the case class) move from rawls-core to rawls-model
+          val submissionId = mapper.readTree(response).get("submissionId").asText()
+
+          // wait for submission to complete
+          Submission.waitUntilSubmissionComplete(projectName, workspaceName, submissionId)
+
+          // verify submission status is done
+          val expectedSubmissionStatus = "Done"
+          val actualSubmissionStatus = Submission.getSubmissionStatus(projectName, workspaceName, submissionId)
+          withClue(s"Submission $projectName/$workspaceName/$submissionId status should be $expectedSubmissionStatus") {
+            actualSubmissionStatus shouldBe expectedSubmissionStatus
+          }
+
+          // verify workflows succeeded
+          val getSubmissionUrl  = Uri(Rawls.url).withPath(Path(s"/api/workspaces/$projectName/$workspaceName/submissions/$submissionId"))
+          val submissionResponse = Rawls.parseResponse(Rawls.getRequest(uri = getSubmissionUrl.toString))
+          // TODO: AS-622 parse into 'Submission' case class following its (i.e. the case class) move from rawls-core to rawls-model
+          val workflows: List[JsonNode] = mapper.readTree(submissionResponse).get("workflows").elements().asScala.toList
+          workflows.foreach { workflow =>
+            val expectedWorkflowStatus = "Succeeded"
+            val actualWorkflowStatus = workflow.get("status").asText()
+            withClue(s"Unexpected status: '${actualWorkflowStatus}'") {
+              actualWorkflowStatus shouldBe expectedWorkflowStatus
+            }
```

Comment:
Do we want to check if the analysis results got written to the bucket?

Reply:
Yes, could be worth checking. There are some additional improvements for this test that'll come in AS-622, so I'll look into checking the buckets as part of that ticket.

Reply:
imho this is not worth testing here. The bucket output is a native feature of Cromwell; we're not adding anything to that feature in Rawls or elsewhere - so it's not worth adding a test, since Cromwell already tests it.